### PR TITLE
change notation to ⇀ and ≤ᵗ

### PR DESCRIPTION
**Pull request by williamdemeo**
+ rename _↛_ to _⇀_
+ rename _≥ᵗ_ to _≤ᵗ_

### CODE REVIEW

File: src/Ledger/Prelude.agda [hunk 1]
```diff
@@ -85,5 +85,5 @@ module Properties where
   module _ {A : Set} ⦃ _ : DecEq A ⦄ where
     open Intersectionᵖ {A} ∈-sp public
 
-_ᶠᵐ : {A B : Set} → A ↛ B → FinMap A B
+_ᶠᵐ : {A B : Set} → A ⇀ B → FinMap A B  -- unicode: `\rightharpoonup`
```
Comment (WhatisRT) [inline src/Ledger/Prelude.agda:88]: I don't think we should leave notes on how to type something in random comments

**Review by WhatisRT — CHANGES_REQUESTED** [review]
This is nice, but there are a bunch of new places with maps in the Voltaire branch. I'd rather change the arrow once that's merged. So we can either leave this open until then, or you can just change the inequality symbol and we merge it now.

File: src/Ledger/Prelude.agda (hunk 1 above)
Comment (williamdemeo) [reply src/Ledger/Prelude.agda:88]: Okay, let's wait.

**Comment by williamdemeo** [comment]
Okay, let's wait.

> This is nice, but there are a bunch of new places with maps in the Voltaire branch. I'd rather change the arrow once that's merged. So we can either leave this open until then, or you can just change the inequality symbol and we merge it now.

**Comment by williamdemeo** [comment]
The change to harpoon notation for partial functions is already part of #105.  The leq notation is too minor for a PR and should be incorporated in a later cosmetic/style PR.

**Pull request closed without merging**